common/workunit/workunit.hpp

```diff
@@ -1748,13 +1748,16 @@ inline cost_type calcCost(cost_type ratePerHour, unsigned __int64 ms) { return r
 extern WORKUNIT_API void executeThorGraph(const char * graphName, IConstWorkUnit &workunit, const IPropertyTree &config);
 
 #ifdef _CONTAINERIZED
+enum KeepK8sJobs { none, podfailures, all };
```

Comment:
should possibly use "enum class" to prevent none/all being defined in scope.

Reply:
yes that would be better, changed.